docker-compose-dev.yml

```diff
@@ -1,34 +1,49 @@
 version: '3.9'
 
 services:
-  frontend:
+  frontend-scoring:
     build:
-      context: ./frontend
+      context: ./frontend_scoring
       dockerfile: Dockerfile-dev
     restart: always
+    ports:
+      - 3000:5173
```

Comment:
Not need to do this, when we develop with 3000 port, we won't be able to connect to backend.